aerosol_dynamics/particle_test.py

```diff
@@ -27,13 +33,30 @@
 
 
 def test_getters():
+    """
+    Test that the getters work.
+    TODO:
+    - can we check units instead of numbers?
+    """
+
     assert small_particle.name() == "small_particle"
     assert small_particle.radius() == 1.0e-9 * u.m
-    assert small_particle.mass() == (4.0/3.0 * np.pi * (1.0e-9 ** 3 * u.m ** 3) * (1.0 * u.kg / u.m**3))
+    assert small_particle.mass() == (
+        4.0/3.0 * np.pi * (1.0e-9 ** 3 * u.m ** 3) * (
+            1.0 * u.kg / u.m**3
+        )
+    )
     assert small_particle.charge() == 1.0
-  
+
 
 def test_knudsen_number():
+    """
+    Test that the knudsen number is calculated correctly.
+    TODO:
```

Comment:
We check units using the decorators in the particle file, so there is no need to test here.
If you feed in the wrong units, it will throw an error at the decorator.
`@u.wraps(u.m, [None])` will only output in meters
Knudsen number is dimensionless, so there's no need to check for units

Reply:
I see, thanks for explaining. We will want to refine these tests as we go on, but I now understand how you're doing it more fully.